fix(reset) return to login after resetting password

edlerd opened the pull request:
# Done
- include return to when starting reset password flow from login


fixes #366

nsklikas reviewed (commented):
Can you add an e2e test for this? Uncommenting https://github.com/canonical/identity-platform-login-ui/blob/main/ui/tests/reset-password.spec.ts#L29 should be enough.

edlerd commented:
> Can you add an e2e test for this? Uncommenting https://github.com/canonical/identity-platform-login-ui/blob/main/ui/tests/reset-password.spec.ts#L29 should be enough.

Good hint, thanks! I updated the test.

How nice of the e2e test suite: It surfaced a missing piece, that is now fixed. I also updated one of the screenshots from the reset password flow, because indeed that step changed :)

nsklikas reviewed (approved):
Nice!! LGTM

natalian98 reviewed (approved):
lgtm, thanks for the fix
Could you rebase the PR?

edlerd commented:
Thanks for the reviews @natalian98 and @nsklikas 
Just pushed the rebase on main.
